Library/Homebrew/cmd/cat.rb

```diff
@@ -25,6 +25,6 @@ def cat
     raise "`brew cat` doesn't support multiple arguments" if args.remaining.size > 1
 
     cd HOMEBREW_REPOSITORY
-    safe_system "cat", formulae.first.path, *ARGV.options_only
```

Comment:
Meant to comment on this before: can the `ARGV.options_only` method be removed entirely now? If so and if it has no usage in Homebrew/homebrew-core: please remove it. Thanks!

Reply:
It has one usage in `formula_installer.rb`

Reply:
Cool. What's blocking that removal?

Reply:
`formula_installer.rb` and `build.rb` directly read and modify `ARGV` https://github.com/Homebrew/brew/issues/5730#issue-409874027